concent_api/common/fields.py

```diff
@@ -60,3 +60,10 @@ class ChoiceEnum(enum.Enum):
     @classmethod
     def choices(cls):
         return tuple((x.name, x.value) for x in cls)
+
+
+class IntegerEnum(enum.Enum):
```

Comment:
Why creating own class instead of using `enum.IntEnum` (https://docs.python.org/3/library/enum.html#enum.IntEnum)?

Reply:
@rwrzesien, cannot we just use `enum.IntEnum` (https://docs.python.org/3/library/enum.html#enum.IntEnum)?

Reply:
@dybi Then we will have to call extra function on each usage, this way we are avoiding it.

Reply:
@rwrzesien, sorry I do not get it. What function we would need to call and why?

Reply:
@dybi `choiceadapter()` in each fields initialization, you should see how it was done previously by @pawelkisielewicz here.

Reply:
Ok, so our `IntegerEnum` should inherit from `IntEnum` -> this way we will ensure that values are only of type `int`